Source/Core/Core/HW/EXI/BBA/BuiltIn.h

```diff
@@ -77,3 +92,22 @@ struct StackRef
   BbaTcpSocket tcp_socket;
   u64 poke_time;
 };
+
+// Max 10 at same time, I think most gc game had a
+// limit of 8 in the GC framework
+using StackRefs = std::array<StackRef, 10>;
+
+class NetworkRef
+{
+public:
+  StackRefs& data();
```

Comment:
since these are super basic, it's probably fine to implement these inline.

Also it might be nice to also provide const and non-const `begin()`/`end()` member functions so you can iterate without needing to get the iterator off `data()`, which would be a little more idiomatic.